YogaKit/Tests/YogaKitTests.m

```diff
@@ -384,4 +385,172 @@ - (void)testThatANonLeafNodeCanBecomeALeafNode
     [container.yoga applyLayout];
 }
 
+- (void)testPositionalPropertiesWork
+{
+  UIView *view = [[UIView alloc] initWithFrame:CGRectZero];
+
+  view.yoga.left = 1;
+  XCTAssertEqual(YGNodeStyleGetPosition(view.yoga.node, YGEdgeLeft).value, 1);
```

Comment:
Thanks for clarifying!

I was just thinking that if we had a unit test that tested layout while using margin/paddng/border, etc, it would become clear that the `NodeRef`'s weren't being properly set because the layout would be in correct. Since this accomplishes the same thing (yet in a slightly more verbose way) I won't argue the point anymore :) Thanks for the test cases!